apps/spruce/src/pages/adminSettings/index.tsx

```diff
@@ -14,12 +14,18 @@ import {
   getAdminSettingsRoute,
   AdminSettingsTabRoutes,
 } from "constants/routes";
+import { type AdminSettings, BannerTheme } from "gql/generated/types";
```

Comment:
nit. can omit `type`
```suggestion
import { AdminSettings, BannerTheme } from "gql/generated/types";
```